### DISCUSSION TITLE
Remove tidyverse from Suggests

### DISCUSSION THREAD

**Discussion by agila5** [luukvdmeer/sfnetworks#139] [Debate]
I noticed that, at the moment, the package_Suggests_ `tidyverse` since it's loaded in several vignettes. I think that we could rewrite a small part of those vignettes and remove the (generic) `tidyverse` dependency since it implies hundreds of useless packages are installed during GHA.

**Comment by Robinlovelace** [comment]
Sounds reasonable to me.

**Comment by luukvdmeer** [comment]
I like to use `library(tidyverse)` in the vignettes such that we can use functions from all tidyverse packages inside examples, without having to explicitly load each individual one. Often I don't even remember if a functions comes from `tidyr` or `dplyr`. But am open to have it changed if there is too much unneccessary package loading going on.

**Comment by Robinlovelace** [comment]
Pros and cons.

Pro: you have the whole tidyverse in there for vignettes.

Con: slower to install on CI (but shouldn't be any slower for most users to install as Suggests are not installed by default).

**Comment by luukvdmeer** [comment]
I got an email from Hadley politely requesting to remove tidyverse from suggests, so I did that now :)
